includes/Widgets/StoreContactForm.php

```diff
@@ -13,17 +13,30 @@
  */
 class StoreContactForm extends WP_Widget {
 
+    /**
+     * Google reCaptcha site key
+     *
+     * @since 3.3.2
+     *
+     * @var string|boolean
+     */
+    public $recaptcha_site_key;
```

Comment:
Are we using the scope of this attribute publicly? If not, consider making it protected.